operator/cmd/mesh/testdata/manifest-generate/output/gateways_override_default.golden.yaml

```diff
@@ -797,7 +797,7 @@ spec:
 apiVersion: policy/v1beta1
 kind: PodDisruptionBudget
 metadata:
-  name: ingressgateway
+  name: istio-ingress-1
```

Comment:
Why does this end with “-1”

Reply:
That's the gateway name in the input gateways_override_default.yaml

Reply:
Thanks.